src/test/java/test/integration/ContainerInstallationIntegrationTest.java

```diff
@@ -272,12 +262,11 @@ private void installContainerAssertProfileAndDependencies(final String container
     }
 
     private void executeCmd(String container) throws TimeoutException {
-        shell().execute("arquillian-setup --container-adapter " + container + " --test-framework junit");
+        shell().execute("arquillian-setup --container-adapter " + container + " --test-framework junit", 60);
```

Comment:
In general I don't feel happy with the way how it's all extracted to multiple reusable methods. We should probably think about introducing parametrized tests instead and have one or two tests which says exactly what's going on. Otherwise it's hopping between to many methods.

60s timeout because for some reason jboss-5 and 6 are talking long even on my machine.